NEWS.md

```diff
@@ -0,0 +1,150 @@
+This file contains a summary of important user-visible changes.
+
+cvc5 1.0
+=========
+
+**New Features**
+
+* *Streamlined C++ API*
+
+* *Two new Python language bindings*
+  - Base module: Feature complete with C++ API
+  - Pythonic module: More pythonic bindings based on the base module
+
+* *New Java language bindings*
+
+* *Theory of Bags (Multisets)*
+  - TODO: describe Bags
```

Comment:
```suggestion
  A new theory of bags which are sets that allow duplicates. It supports basic operators like union disjoint, union max, intersection, difference subtract, difference remove, duplicate removal, and multiplicity of an element in a bag. 
```

Reply:
Reworded to "collections" to be generic (they are not sets).